pna_control/pna_control.py

```diff
@@ -114,7 +136,40 @@ def powersweep(startpower: float, endpower: float, numsweeps: int, centerf: floa
     file.write('POINTS: '+str(points)+'\n')
     file.close()
 
-    #run each sweep while increasing averages for each power
+    #run each sweep
     for i in sweeps:
-        getdata(centerf, span, temp, averages, i, edelay, ifband, points, outputfile)
+        get_data(centerf, span, temp, averages, i, edelay, ifband, points, outputfile)
+        # Iterate number of averages as power decreases
         averages = averages * ((10**(stepsize/10))**0.5)
+
+def name_datafile(outputfile,power,temp):
+    filename = outputfile+'_'+str(power)+'dB'+'_'+str(temp)+'mK'
+    filename = filename.replace('.','p')
+
+    return filename
+    
+def timestamp_folder(dir,meastype):
+    result = time.localtime(time.time())
```

Comment:
A doctstring would be really helpful here as this is doing a couple things, and I had to run the function to figure out what it was doing. Something like:
```python
"""Create a filename and directory structure to annotate the scan.

Takes a root directory, appends scan type and timestamp.

Args:
    dir: root directory for the scan
    meastype: type of measurements, eg: 'powersweep' 

Returns:
    Formatted oath eg. foo/bar_202007013_9_45_28/ 
"""
```